src/iuse.cpp

```diff
@@ -3072,6 +3146,8 @@ int iuse::noise_emitter_on(player *p, item *it, bool t, const tripoint &pos)
 
 int iuse::ma_manual(player *p, item *it, bool, const tripoint& )
 {
+    // [CR] - should NPCs just be allowed to learn this stuff? Just like that?
```

Comment:
Same as players, I'd like to change it to a slow process of learning a style, so just getting the book wouldn't make you go, "I know kung-fu".  But that's obviously future stuff, right now you just read it and bam, bruce lee.